### ISSUE TITLE
Accessibility improvements for chat notifications

### ISSUE THREAD

**Issue by ajesina** [bigbluebutton/bigbluebutton#10942]
**Describe the bug**
When a chat panel is already open, no notification sounds are being played for any other chat. This is an issue particularly for screen reader users, since they will not get informed about new messages unless they additionally enable push notifications.
Additionally, it might be helpful to have a separate sound for private and public chat notifications.

**To Reproduce**
- Join a session and enable chat audio notifications
- Open the public chat
- Join the session with a second user and send a private message to the first user

**Expected behavior**
A notification sound for the private chat should be played.

**Actual behavior**
No notification sound is played.

**Comment by KDSBrowne** [comment]
I'm not sure a separate sound is necessary for private chats.
From my tests it seems that the notification sound is indeed being played for new
chat messages.

As for the screen readers, they also inform the user already about new messages without
additionally enabling push notification. See gif below: 

![SR_chat_msgs](https://user-images.githubusercontent.com/22058534/147572759-efdd2019-f8d2-45be-94be-a254e11064b8.gif)

The Issue with the screen readers is that there
is no announcement for each new message notification when the users list is closed (fixed in #13986).
